### DISCUSSION TITLE
Unknown meeting, unknown address

### DISCUSSION THREAD

**Discussion by boboinmo2021** [code4recovery/12-step-meeting-list#829] [Q&A]
I have a persnickety problem with only one meeting. I think a bad address was inserted by someone and its now stuck in the system and now it autocorrects to the bad address. There is no direct way to edit locations once they are in. Could this be done with a mysql editor? Or is there a way to delete the location record altogether.  

I got it to work but i had to give it a slightly off address otherwise it constantly corrects it to a bad address of: 3115 W, 3115 Elm St, St Charles, MO 63301, USA
This is what works but the address is incorrect. 
MO St Charles, 8:00 pm, (X,ONL) Group 777. Good Shepard
United Church of Christ. 3117 Elm St. Rotating Meeting Zoom
id: 853-050-3191.

If I snipe in an try to do remove the 3115 W. or just change to 3115 Elm St the "update" button greys out till it updates the field to the bad address.

**Comment by joshreisner** [comment]
i think this is happening because this is what google thinks the address is. if the pin is in the right place i wouldn't worry about it.

<img width="468" alt="Screen Shot 2022-06-13 at 11 52 13 AM" src="https://user-images.githubusercontent.com/1551689/173431282-8a38b544-bfaf-4f32-86c6-8f56520fd1d4.png">

if you want to add code to your theme it is possible to override google's definition with your own: https://wordpress.org/plugins/12-step-meeting-list/#i%20need%20to%20correct%20a%20meeting%20address%20or%20change%20a%20pin%27s%20location

**Reply by joshreisner** [reply]
i think this should work…

```
if (function_exists('tsml_custom_addresses')) {
    tsml_custom_addresses(array(
        '3115 W, 3115 Elm St, St Charles, MO 63301, USA' => array(
            'formatted_address' => '3115 W Elm St, St Charles, MO 63301, USA',
            'city' => 'St Charles',
            'latitude' => 38.8050199,
            'longitude' => -90.5084694,
            'approximate' => 'no',
        ),
    ));
}
```

**Reply by boboinmo2021** [reply]
How do I add this?

**Reply by boboinmo2021** [reply]
I am adept and unafraid but need the guidance

**Reply by joshreisner** [reply]
we typically advise people to add this to their theme's functions.php (or child theme, if your theme is a third party theme)

be careful! might be good to make a quick backup before doing anything

**Comment by boboinmo2021** [comment]
It is but it shows up in the pdf generator as "Unknown meeting, unknown address” instead of the meeting address and name.

On Jun 13, 2022, at 2:39 PM, Josh Reisner ***@***.******@***.***>> wrote:



i think this is happening because this is what google thinks the address is. if the pin is in the right place i wouldn't worry about it.

[Screen Shot 2022-06-13 at 11 52 13 AM]<https://user-images.githubusercontent.com/1551689/173431282-8a38b544-bfaf-4f32-86c6-8f56520fd1d4.png>

if you want to add code to your theme it is possible to override google's definition with your own: https://wordpress.org/plugins/12-step-meeting-list/#i%20need%20to%20correct%20a%20meeting%20address%20or%20change%20a%20pin%27s%20location

**Comment by boboinmo2021** [comment]
I will for sure. Thank you

On Jun 13, 2022, at 16:51, Josh Reisner ***@***.***> wrote:

﻿

we typically advise people to add this to their theme's functions.php (or child theme, if your theme is a third party theme)

be careful! might be good to make a quick backup before doing anything
